### PR TITLE
New README

### PR DESCRIPTION
**Pull request by safinsingh**
Add more descriptive README

### CODE REVIEW

**Comment by PgBiel** [comment]
We should probably wait for a decision regarding LaTeX distributions before we merge this PR

File: README.md [hunk 1]
```diff
@@ -8,102 +8,197 @@
...
-## Anaconda Install
+### Installing Manim-Community itself
 
-* Install sox and latex as above. 
-* Create a conda environment using `conda env create -f environment.yml`
-* **WINDOWS ONLY** Install `pyreadline` via `pip install pyreadline`. 
+Manim runs on Python 3.7+. If you'd like to just use the library, you can install it from PyPI via pip:
 
+```sh
+pip3 install manimlib
```
Comment (eulertour) [inline README.md:161]: Does this work without a setup.py?

File: README.md [hunk 2]
```diff
@@ -8,102 +8,197 @@
...
+## Table of Contents:
+- [Installation](#installation)
+  * [Windows](#windows)
+    + [Cairo Installation](#cairo-installation)
+    + [FFmpeg Installation](#ffmpeg-installation)
+    + [SoX Installation](#sox-installation)
+    + [LaTeX Installation](#latex-installation)
+  * [Linux](#linux)
+  + [Cairo Installation](#brew-installation)
+    + [Cairo Installation](#cairo-installation-1)
```
Comment (eulertour) [inline README.md:22]: Should this only be here once?

File: README.md (hunk 1 above)
Comment (PgBiel) [reply README.md:161]: I don't think so

**Comment by eulertour** [comment]
I don't see why this couldn't be merged now, it's a great improvement over the old README and we can always change it later.

**Comment by PgBiel** [comment]
I just wanted to take a better look at it, which will take a bit because I'm about to sleep... but sure, we can do that. If I have any complaints we can deal with it later

File: README.md (hunk 1 above)
Comment (eulertour) [reply README.md:161]: Then this should be removed first.

Comment (safinsingh) [reply README.md:161]: I believe someone in the discord said to keep this here, should I just remove it?

File: README.md (hunk 2 above)
Comment (safinsingh) [reply README.md:21]: My bad, this is a mistake, let me fix it...

Comment (safinsingh) [reply README.md:21]: Nvm, Aathish already did

File: README.md (hunk 1 above)
Comment (eulertour) [reply README.md:161]: It's fine then, we should have a setup.py shortly

**Pull request merged**